lab-ron/test/read-files-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const assert = require('assert');
+const readFileHelper = require('../lib/read-files.js');
+
+describe('read-files.js', function(){
```

Comment:
@Rubiksron best practice is to describe this as the module or method as opposed to the filename - for example:

you have:
```
describe('read-files.js', function() {  // test stuff here });
```
try something similar to this (anything works, as long as it's descriptive and isn't the filename):
```
describe('File Reader', function() { // do stuff here });
```